Documentation/README.md

```diff
@@ -20,6 +20,7 @@ you are welcome to ask on [Discord](../README.md#get-in-touch-and-participate).
 * [Qt Creator](EditorConfiguration/QtCreatorConfiguration.md)
 * [Vim](EditorConfiguration/VimConfiguration.md)
 * [VS Code](EditorConfiguration/VSCodeConfiguration.md)
+* [VS Code Dev Container](EditorConfiguration/VSCodeDevContainer.md)
```

Comment:
hm. Technically, the devcontainer is not vscode specific. several IDEs have support. Though we do have vscode-specific config in there for it to use.

Reply:
Yeah, both settings and extensions are vscode-specific. I guess only vscode currently gives the ready-to-use container experience. Should we leave this one unchanged till the developer that uses the container in another editor shows up?